web/modules/custom/rsvplist/src/Form/RSVPForm.php

```diff
@@ -7,6 +7,7 @@
 
 /**
  * @package \Drupal\rsvplist\Form\RSVPForm
+ *   RSVP Form Class.
  */
```

Comment:
This is incorrect way of writing the class docblock.
Please fix all the other class docblocks like this.

Reply:
ok sir